extension/chrome/elements/composer/composer-pwd-or-pubkey-container.ts

```diff
@@ -52,9 +72,20 @@ export class ComposerPwdOrPubkeyContainer extends ComposerComponent {
     this.composer.size.setInputTextHeightManuallyIfNeeded();
   }
 
-  private showMsgPwdUiAndColorBtn = () => {
-    this.composer.S.cached('password_or_pubkey').css('display', 'table-row');
-    this.composer.S.cached('password_or_pubkey').css('display', 'table-row');
+  private showMsgPwdUiAndColorBtn = async () => {
+    if (this.composer.S.cached('password_or_pubkey').is(':hidden')) {
+      let messageExpire: number = 3;
+      try {
+        const authInfo = await Store.authInfo(this.view.acctEmail);
+        const response = await Backend.accountGetAndUpdateLocalStore(authInfo);
+        messageExpire = response.account.default_message_expire;
+      } catch (e) {
+        // ignore, use the default 3 days message expire
+      }
+      const pluralize = (count: number, noun: string, suffix = 's') => `${count} ${noun}${count > 1 ? suffix : ''}`; // TODO: where to move this helper?
```

Comment:
@tomholub Where to move this helper?

Reply:
I guess you could put it in `Str.plularize`, although I would not mind if it just said `for 1 days` since I suspect very few people will pay to set it lower.

(I just checked the backend, it's 4 people, I would not worry about it)

Reply:
Done :+1: